ui/src/graphql/index.ts

```diff
@@ -1,59 +1,29 @@
 export { useCollectionDetails } from './queries/collection_details.ts';
-export type {
-  CollectionDetailsVariables,
-  CollectionDetailsResult,
-} from './queries/collection_details.ts';
 
 export { useCollectionWithColumnDetails } from './queries/collection_with_column_details.ts';
-export type {
-  CollectionWithColumnDetailsVariables,
-  CollectionWithColumnDetailsResult,
-} from './queries/collection_with_column_details.ts';
 
 export { useCollectionInfoForCurrentSession } from './queries/collection_info_for_current_session.ts';
-export type {
-  CollectionInfoForCurrentSessionVariables,
-  CollectionInfoForCurrentSessionResult,
-  CollectionInfoResult,
-} from './queries/collection_info_for_current_session.ts';
 
 export { useUsers } from './queries/users.ts';
-export type { UsersResult } from './queries/users.ts';
 
 export {
   useColumnDefInputValidation,
   useLazyColumnDefInputValidation,
 } from './queries/column_def_input_validation.ts';
-export type {
-  ColumnDefInputValidationVariables,
-  ColumnDefInputValidationResult,
-} from './queries/column_def_input_validation.ts';
 
 export {
   useCollectionDefInputValidation,
   useLazyCollectionDefInputValidation,
 } from './queries/collection_def_input_validation.ts';
 
-export { useCollumnDefCreation } from './mutations/column_def_creation.ts';
-export type {
-  CollumnDefCreationVariables,
-  CollumnDefCreationResult,
-} from './mutations/column_def_creation.ts';
+export { useColumnDefCreation } from './mutations/column_def_creation.ts';
+export type {} from './mutations/column_def_creation.ts';
```

Comment:
Leftover `export type {} ...` lines in this file for cleanup.

Reply:
Cleaned up exports for removed types.